scripts/check-events

```diff
@@ -22,6 +22,10 @@ EVENT_CLASSES = {
   ],
 }
 
+EVENT_NAME_EVENTS_DROPPED = 'component_discarded_events_total'
+EVENT_NAME_ERROR = 'component_errors_total'
```

Comment:
Hmm, I know what you mean, but these are actually the names of counters or, more generically, metrics, not events.

Reply:
Ah, good call. I'll update that. It's easier to read too!